### ISSUE TITLE
False positive on `attributes` rule

### ISSUE THREAD

**Issue by matheusalano** [realm/SwiftLint#4596]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

When we add a comment between the attribute and the variable, it triggers the `attributes` rule.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 
 0.50.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? 
 Homebrew
* Paste your configuration file:

```yml
opt_in_rules:
  - attributes
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
 No
* Which Xcode version are you using (check `xcodebuild -version`)?
 Build version 14B47b
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
extension Property {
    @available(*, unavailable, renamed: "isOptional")
    // Hello world
    public var optional: Bool { fatalError() }
}
```

**Comment by jpsim** [comment]
Why not put the comment above the attribute?
